README.md

```diff
@@ -107,9 +107,6 @@ cd langfuse
 
 # Run server and database
 docker compose up -d
-
-# Apply db migrations
-DATABASE_URL=postgresql://postgres:postgres@localhost:5432/postgres DIRECT_URL=postgresql://postgres:postgres@localhost:5432/postgres npx prisma migrate deploy
```

Comment:
We also need to adjust the docs then i assume

Reply:
Yes. Will do once we merge.